spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+---
+cloudfoundry:
```

Comment:
Also, see my above comment related to putting all those keys in an appropriate namesapce yet to be discussed. cc @markfisher